### ISSUE TITLE
Support pandas options when displaying metas

### ISSUE THREAD

**Issue by epizut** [dask/dask#9096]
Displaying a dask dataframe with a lot of columns in a Jupyter notebook can be very time, memory and space (ie: scroll) consuming for the browser. By default, all columns of the meta are shown, unlike regular pandas df.

It would be useful to implement a few pandas [options](https://pandas.pydata.org/docs/user_guide/options.html) when displaying the meta
- min,max_rows
- min,max_columns
- etc

![image](https://user-images.githubusercontent.com/1694892/168864110-678e64e3-7913-4d0d-b438-d7fa01c9ee04.png)

**Comment by pavithraes** [comment]
@epizut Thanks for opening this, it does like a nice feature to have. :)

**Comment by epizut** [comment]
Agreed, but I am not sure replicating and tracking pandas config changes worth it.
Can we simply let pandas render the ._meta with a little sugar like npartitions, name...
